src/languages/en.ts

```diff
@@ -1667,6 +1667,8 @@ export default {
                 `You have been invited to ${workspaceName || 'a workspace'}! Download the Expensify mobile app at use.expensify.com/download to start tracking your expenses.`,
         },
         editor: {
+            descriptionInputLabel: 'Workspace description',
+            descriptionInputHelpText: 'This is a brief description of your workspace.',
```

Comment:
I don't think this is used anywhere?

Reply:
@jjcoffee It is being used in `WorkspaceDescriptionPage.tsx`

Reply:
@esh-g I see `descriptionInputLabel` being used there but not `descriptionInputHelpText`.

Reply:
Sorry, fixed here: https://github.com/Expensify/App/pull/36058/commits/ea1d39f44d3c4db08845b26fd2dd6f9ff6bc3c98